Config/instanceConfigurator/src/org/akvo/flow/InstanceConfigurator.java

```diff
@@ -76,6 +76,8 @@ public static void main(String[] args) throws Exception {
         String gaeId = cli.getOptionValue("gae");
         String outFolder = cli.getOptionValue("o");
         String flowServices = cli.getOptionValue("fs");
+        String eventNotification = cli.getOptionValue("en");
+        String enableChangeEvents = cli.getOptionValue("ce");
```

Comment:
Do we want to fallback this option to `false` if the user doesn't pass any value? its optional parameter